sonic_data_client/events_client.go

```diff
@@ -97,10 +110,28 @@ func NewEventClient(paths []*gnmipb.Path, prefix *gnmipb.Path, logLevel int) (Cl
         for k, v := range keys {
             if (k == PARAM_HEARTBEAT) {
                 if val, err := strconv.Atoi(v); err == nil {
+                    if (val > HEARTBEAT_MAX) {
```

Comment:
If heartbeat is set to 0, does that mean no heartbeat will be sent? Or will we keep publishing heartbeat events constantly? Wondering about the minimum value of this hearbeat

Reply:
From swsscommon/common/events_wrap.h

/*
 *  Global configurable options can be set via this API.
 *
 *  The options are provided as JSON object with key/values as JSON string.
 *  key = <option name case insensitive>
 *  value = <option value>
 *
 *  Supported options
 *
 *  Option name: HEARTBEAT_INTERVAL_SECS
 *  Option Value: interval in seconds as int
 *      A value of -1 implies no heartbeat
 *      A value of 0 implies the lowest possible interval as possible/supported.
 *          This depends on implementation.
 *      Any non zero value implies count of seconds.
 *          NOTE: System will round it to the multiple of lowest interval supported.
 *      Any negative value other than -1 is treated as invalid.
 *
 *